docs/changelog/104683.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 104683
+summary: "Feature: re-arrange document id bytes to take advantage of shared prefixes"
+area: TSDB
```

Comment:
Is it the right area, I thought it would not apply to TSDB since TSDB uses its own IDs and not auto-generated IDs? (I may be outdated, feel free to correct me!)

Reply:
Should have used the other label indeed :StorageEngine/Logs...definitely not TSDB...TSDB does not use this scheme for doc _id generation.